ios/FluentUI/People Picker/ContactCollectionView.swift

```diff
@@ -0,0 +1,139 @@
+        return heightConstraint
+    }()
+
+    private func configureCollectionView() {
+        translatesAutoresizingMaskIntoConstraints = false
+        showsHorizontalScrollIndicator = false
+        showsVerticalScrollIndicator = false
+        backgroundColor = Colors.surfacePrimary
+        dataSource = self
+        delegate = self
+        contentInset = UIEdgeInsets(top: 0, left: Constants.leadingInset, bottom: 0, right: 0)
+    }
+
+    @objc private func setupHeightConstraint() {
+        let height = (contactList.count > 0) ? UIApplication.shared.preferredContentSizeCategory.contactHeight : 0
+        heightConstraint.constant = height
+    }
+}
+
+extension ContactCollectionView: UICollectionViewDataSource {
+    @objc public func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return contactList.count
+    }
+
+    @objc public func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: ContactCollectionViewCell.identifier, for: indexPath) as! ContactCollectionViewCell
+        cell.setup(contact: contactList[indexPath.item])
+
+        return cell
+    }
+}
+
+extension ContactCollectionView: UICollectionViewDelegate {
+    // If the Contact is not fully visible in the scroll frame, scrolls the view by an offset large enough
+    // so that the tapped Contact is fully visible.
+    @objc public func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        guard let cell = cellForItem(at: indexPath) else {
+            return
+        }
+
+        let cellWidth = cell.frame.width
+        let cellFrame = cell.contentView.convert(cell.bounds, to: self)
+        let cellLeftPosition = cellFrame.origin.x
+        let cellRightPosition = cellLeftPosition + cellWidth
+        let viewLeadingPosition = bounds.origin.x
+        let viewTrailingPosition = viewLeadingPosition + frame.size.width
+
+        let extraScrollWidth = layout.minimumLineSpacingForSectionAt(section: 0)
+        var offSet = contentOffset.x
+        if cellLeftPosition < viewLeadingPosition {
+            offSet = cellLeftPosition - extraScrollWidth
+            offSet = max(offSet, -Constants.leadingInset)
+        } else if cellRightPosition > viewTrailingPosition {
+            let maxOffsetX = contentSize.width - frame.size.width + extraScrollWidth
+            offSet = cellRightPosition - frame.size.width + extraScrollWidth
+            offSet = min(offSet, maxOffsetX)
+        }
+
+        if offSet != contentOffset.x {
+            setContentOffset(CGPoint(x: offSet, y: contentOffset.y), animated: true)
+        }
+    }
+}
+
+extension UIContentSizeCategory {
+    var contactHeight: CGFloat {
+        switch self {
+        case .extraSmall:
+            return ContactCollectionView.Constants.extraSmallContentContactHeight
```

Comment:
We shouldn't use constants in this case. Just return the literal CGFloats.
You could consider var contactHeight as a group of constants.
In other words, remove these constants:
        static let extraSmallContentContactHeight: CGFloat = 115.0
        static let smallContentContactHeight: CGFloat = 117.0
        static let mediumContentContactHeight: CGFloat = 118.0
        static let largeContentContactHeight: CGFloat = 121.0
        static let extraLargeContentContactHeight: CGFloat = 125.0
        static let extraExtraLargeContentContactHeight: CGFloat = 129.0
        static let extraExtraExtraLargeContentContactHeight: CGFloat = 135.0

and hardcode them in here.